src/components/profile.ts

```diff
@@ -156,6 +163,15 @@ export const validUserCustomization = (
         return { reason: 'Invalid year. Must be between: ' + yearStart + ' and ' + yearEnd + '.' };
       }
       break;
+    case validatedFields.profile_emoji:
+      console.log('profile emoji: ' + parsedDescription);
```

Comment:
Remove this?

Reply:
Removed it. Sorry I forgot I was checking this there.